adot/python/src/otel/otel_sdk/otel-instrument

```diff
@@ -44,7 +44,7 @@ environ["OTEL_RESOURCE_ATTRIBUTES"] = "%s,%s" % (
 configured_propagators = environ.get("OTEL_PROPAGATORS")
 if not configured_propagators:
     environ["OTEL_PROPAGATORS"] = "xray"
-elif "aws_xray" not in configured_propagators.split(","):
```

Comment:
Hmm - in other languages, we don't have this behavior of forcing in the xray propagator if the user set a propagator.

https://github.com/aws-observability/aws-otel-lambda/blob/main/java/scripts/otel-handler#L5

https://github.com/aws-observability/aws-otel-lambda/blob/main/nodejs/wrapper-adot/src/adot-extension.ts#L12

I feel this is better since in the end users always need to be allowed a choice, but currently it's not possible to disable the xray propagator no matter what.

Also good to have the default include w3c and b3 as the others